### ISSUE TITLE
Allow Markdown and LaTeX/MathJax for project content

### ISSUE THREAD

**Issue by matthewreyna** [MIT-LCP/physionet-build#1390]
Project maintainers currently use a WYSIWYG editor and/or HTML source code to add and maintain project content.

I would like for the site to allow users to use Markdown and LaTeX/MathJax to add and maintain project content. As you know, these are common, plain text formats. A suitable version of Markdown, such as CommonMark, and a limited version of LaTeX could be used.

Most likely, this is not an easy change, but this change will allow project maintainers to better add and maintain project content, and it will make the site more powerful. For example, search should be more effective if you can search the Markdown source instead of (or in addition to) the HTML source.

**Comment by tompollard** [comment]
I'd love to support markdown. See also: https://github.com/MIT-LCP/physionet-build/issues/371.  Side note, but the current editor does currently support LaTeX for math.

**Comment by matthewreyna** [comment]
The current editor does support LaTeX, but each equation must be added separately through a WYSIWYG interface. I think that this makes it harder to add and maintain mathematical content, and some equations on the site have errors.

More specifically, I am requesting support for Markdown with (a subset of) LaTeX interwoven as plain text, much like GitHub Pages and Jupyter Notebooks support.

**Comment by tompollard** [comment]
> More specifically, I am requesting support for Markdown with (a subset of) LaTeX interwoven as plain text, much like GitHub Pages and Jupyter Notebooks support.

Thanks Matthew, I agree this would be nice.

**Comment by tompollard** [comment]
Yes, definitely helpful! We're limited on what kind of development work we can do at the moment, but it's good to track bugs, suggestions, etc here, and at some point I hope we can address them.
